inspirehep/dojson/common/base.py

```diff
@@ -216,10 +217,13 @@ def collections(self, key, value):
 
     def get_value(value):
```

Comment:
I tested an example with `timeit` and it seems that this way of writing code is not that much slower. For 1M function calls the difference was two seconds.